package/CHANGELOG

```diff
@@ -81,6 +81,8 @@ Changes
  * Removed undocumented and unused attribute
    `analysis.lineardensity.LinearDensity.totalmass` (PR #5007)
  * Remove `default` channel from RTD conda env. (Issue # 5036, PR # 5037)   
+ * Changed the fragment detection logic in `analysis.polymer.sort_backbone` (Issue #5112, PR #5113)
+ * Replaced the while loop in `analysis.polymer.sort_backbone` with a for loop (Issue #5112, PR #5113)
```

Comment:
You are fixing a problem so you just add one entry under Fixes. 

Users just need to know that sort_backbone is now also working for discontinuous polymers and does not result in an infinite loop.

No need to mention internal details such as a while loop – keep it big-picture, user-perspective.

Reply:
Also add your GH handle to the author list for 2.10.0